event/models.py

```diff
@@ -8,8 +10,31 @@ def __str__(self) -> str:
         return self.name
 
 
+class EventLocation(models.Model):
+    name = models.CharField(max_length=150, blank=False, null=False)
+    # it is nullable as the location may be not always findable
+    google_maps_location = models.URLField(
+        max_length=500, blank=True, null=True
+    )
+
+    def __str__(self):
+        return self.name
+
+
+class HotTopic(models.Model):
+    name = models.CharField(max_length=100, primary_key=True)
+
+    def __str__(self):
+        return self.name
+
+
 # Create your models here.
 class Event(models.Model):
+    slug = models.SlugField(
```

Comment:
adding fallback of slug if entered repeatedly will be beneficial
pre_save slug for bulk operation

Reply:
What do you mean ?